pythonparser/ast.py

```diff
@@ -57,6 +57,15 @@ def value(name):
                            self._reprfields()))
         return "%s(%s)" % (self.__class__.__name__, fields)
 
+    # Compat with stdlib
+    @property
+    def lineno(self):
+        return self.loc
```

Comment:
I don't think this is right. From the Python docs:
```
Instances of ast.expr and ast.stmt subclasses have lineno and col_offset attributes. The lineno is the line number of source text (1-indexed so the first line is line 1) and the col_offset is the UTF-8 byte offset of the first token that generated the node. The UTF-8 offset is recorded because the parser uses UTF-8 internally.
```
Conversely, `self.loc` isn't even a number.